Sources/APNSwift/LiveActivity/APNSLiveActivityNotification.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  APNSLiveActivityNotification.swift
+//  PushSender
+//
+//  Created by csms on 20/09/2022.
+//
+
+import struct Foundation.UUID
+
+/// An alert notification.
+///
+/// - Important: Your dynamic payload will get encoded to the root of the JSON payload that is send to APNs.
+/// It is **important** that you do not encode anything with the key `aps`.
+public struct APNSLiveActivityNotification<ContentState: Encodable>: Encodable {
+    enum CodingKeys: CodingKey {
+        case aps
+    }
+
+    /// The fixed content to indicate that this is a background notification.
+    private var aps: APNSLiveActivityNotificationAPSStorage<ContentState>
+
+    // Timestamp when sending notification
```

Comment:
```suggestion
    /// Timestamp when sending notification
```

Is there anything else we can say about what this timestamp does?